src/Tween.js

```diff
@@ -11,12 +11,12 @@ var TWEEN;
 
 (function () {
 
-	var _Group = function () {
+	var Group = function () {
```

Comment:
👍 

Thoughts on #379?

I think I'd prefer to have a single factory function. If you'd like to modify this PR to do similar to my PR, let's get it merged so you can get the credit. 😄

Reply:
@finscn Also note that the commit messages should follow semantic-release format. See here: https://github.com/semantic-release/semantic-release#default-commit-message-format